### DISCUSSION TITLE
Partially install Rook/Ceph cluster

### DISCUSSION THREAD

**Discussion by KKonak** [rook/rook#15154] [Q&A]
Is there a way to partially build the Rook/Ceph cluster? The scenario is: I have a 3 node cluster with 3 monitors and 2 OSDs, in this case I want to create my ceph cluster when only 2/3 nodes are ready/exist. In this case there would still be monitor quorum and both OSDs would exist so Rook would be functional in a degraded state. 

If the 3 nodes are stated in the cephClusterSpec to place monitors on, the installation is stuck because only 2/3 nodes are ready, is there a way around this check? The third node (VM) ONLY runs a monitor to float between server to keep quorum. Once the VM is ready I would expect the third monitor pod/deployment to sit in pending waiting for the desired node.

Kubernetes would handle the hard part of scheduling, I just want to bypass the initial check of 3/3 nodes being ready. It would be something similar to this use case in 1.15.5: https://github.com/rook/rook/pull/14949 Except the node may / may not exist at the time.

**Comment by travisn** [comment]
How much delay are you expecting to bring up the third node? On initial deployment Rook does expect to start all the mons, so I don't see a workaround for this today, other than to initially configure Rook (in the cephCluster CR) with 2 mons and then update it to 3 after the new node is available.
